templates/includes/base.html

```diff
@@ -38,6 +38,18 @@
 
 
 
+        <script src="https://cdn.jsdelivr.net/npm/sweetalert2@11"></script>
+        {% if messages %}
+            {% for message in messages %}
+            <script>
+                Swal.fire({
+                    text: "{{message}}",
+                    icon: "{{message.tags}}",
+                });
+            </script>
+            {% endfor %}
+        {% endif %}
```

Comment:
NO sweetalert2 needed

Reply:
replaced with simple alerts

Reply:
No alerts. Just render on the page.

Reply:
only error are added say what if email already exists or while login password is incorrect